### ISSUE TITLE
Improve split transactions

### ISSUE THREAD

**Issue by simhnna** [agstrike/silverstrike#7]
Currently the form to enter/edit them is rather hard to use.  
I'm not sure yet what exactly the splits should be like.
* Allow different accounts to be involved (for instance income that has tax deductions, etc.)
* Only allow for different titles/categories

**Comment by simhnna** [comment]
The normal split form should only display splits that have a personal account in src.

Foreign splits should just be auto-created, so that transactions are always correct.

**Comment by speedyflipper** [comment]
I frequently have split transactions that have more than two destination accounts from a single source account. 

For example if I go out to dinner with friends and I pay the whole bill. The whole bill will be the source account, however the destination account will be split between the restaurant and each friend with a different category for each.

**Comment by simhnna** [comment]
yeah. I'm currently working on having transactions always be between exactly two accounts.

However I plan to support the ability to include more accounts in the splits that have to balance in such a way that the amount transferred between the two "transaction"-accounts is valid.

I still need to play around a little to see how that actually works...

**Comment by qedi-r** [comment]
Yeah, I too have single transactions in my accounts that should go to different foreign accounts. The most common example I have, is that X dollars comes out of my checking account for insurance to one provider, but x is house and y is auto.

Right now, it seems that multiple foreign accounts kind of already works? There's not really any validation on the splits directly, though. I will likely file this as a separate issue so I can fix it.
